src/input-helper.ts

```diff
@@ -0,0 +1,20 @@
+import * as core from '@actions/core'
+import {Inputs, NoFileOptions} from './constants'
+import {UploadInputs} from './upload-inputs'
+
+/**
+ * Helper to get all the inputs for the action
+ */
+export function getInputs(): UploadInputs {
```

Comment:
`actions/checkout` action has ALOT of available options and it has a [input-helper](https://github.com/actions/checkout/blob/main/src/input-helper.ts) file that is used get everything. I'm copying the same thing. In the future if there are more input options needed for things like retention, this will help out